kotlin/src/main/com/looker/rtl/Transport.kt

```diff
@@ -187,8 +213,29 @@ class Transport(val options: TransportOptions) {
             path: String,
             queryParams: Values = mapOf(),
             body: Any? = null,
-            noinline authenticator: Authenticator? = null): SDKResponse {
+            noinline authenticator: Authenticator? = null) : SDKResponse {
+        // TODO get overrides parameter to work without causing compilation errors in UserSession
```

Comment:
That particular TODO is necessary because I haven't figured out why callers to that method break when I introduce the `overrides` param. I'm hoping to discussing with @brianmadden soon.